frontend/src/app/accountCreation/PasswordForm/PasswordForm.tsx

```diff
@@ -22,15 +26,24 @@ export const PasswordForm = () => {
   const [passwordConfirmationError, setPasswordConfirmationError] = useState(
     ""
   );
+  const [loading, setLoading] = useState(false);
+  const [submitted, setSubmitted] = useState(false);
 
+  // Get activation token from store
+  const activationToken = useSelector((state: any) => state.activationToken);
```

Comment:
This should be able to be typed better `const activationToken = useSelector<RootState, string>((state) => state.activationToken);`